app/mailers/user_mailer.rb

```diff
@@ -218,14 +218,15 @@ def verify_by_mail_letter_requested
     end
   end
 
-  def add_email(token)
+  def add_email(token:, request_id: nil)
```

Comment:
Can we update the `user_mailer_preview.rb` to pass this `request_id` parameter?

I feel making all these properties optional when we don't actually expect them to ever not be passed allows for unintended gaps in test coverage and preview behaviors. Can you check that all the `nil`-able properties you've added actually need to be `nil`-able?

```suggestion
  def add_email(token:, request_id:)
```

Reply:
Removed unneeded nil presupposition.